### DISCUSSION TITLE
12/26 정민채 블로그 3회차

### DISCUSSION THREAD

**Discussion by Jungminchae** [Orm-camp/blog-feedback#54] [동글동글 4기 피드백]
https://chaechae.life/blog/python-web-automation-helium

**Comment by smahn9123** [comment]
특별히 Helium으로 섵택하신 이유 있으신가요?

**Reply by smahn9123** [reply]
전 Playwright 써봤었는데 문서는 잘되있긴한데 크롤링은 그냥 힘든 거 같아요ㅜ
